packages/utilities/exposure-scanning/patch/index.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable global-require */
+let octokit;
+
+// Used to limit the number of calls to the patch comment and review comment endpoint
+const maxCommentPatches = 20;
+
+(async () => {
+  if (process.env.GITHUB_ACTION && process.env.GITHUB_TOKEN) {
+    const { Octokit } = await import('@octokit/action');
+    octokit = new Octokit();
+  } else {
+    const { Octokit } = await import('@octokit/rest');
+    octokit = new Octokit();
+  }
+})();
+
+const patchPrBody = async (reqBody, body) => {
+  await octokit.request('PATCH /repos/{owner}/{repo}/pulls/{id}', {
+    ...reqBody,
+    body,
+  });
+};
```

Comment:
```suggestion
const patchPrBody = (reqBody, body) =>
  octokit.request('PATCH /repos/{owner}/{repo}/pulls/{id}', {
    ...reqBody,
    body,
  });
```
I _think_ following on from @jroebu14 's `async` comments this await may be redundant. As I understand it you will be returning this `Promise` returned by the Octokit api rather than generating a new one by awaiting the `Promise` from Octokit. I'm not completely certain on this, let me know if I'm wrong so I can learn 😅

Reply:
Ahhh good point, yeah I think you're right I'll give it a go 👍

Reply:
This might help - if you want the `patchPrBody` to resolve with an `undefined` value then Harry's current code is what you want. If you want to return whatever value the `octokit.request` promise resolves to then Andrew's suggestion is what you want. If you don't use the resolved value then Andrew's suggestion has the benefit of being terser.

Reply:
Greats thoughts @jroebu14 , I'd actually probably leave it as it is on that basis as it better encapsulation to not return the result from this function; there is no need for the caller to know (or care) that octokit was used to make the request.